services/webhook/update_flows.sh

```diff
@@ -1,5 +1,6 @@
 #!/usr/bin/env bash
 
 git pull origin main
+/home/sglyon/miniconda/envs/prefect-can-scrapers/bin/pip install -r /home/sglyon/can-scrapers/requirements.txt
```

Comment:
The previous change (to add `pip install -e`) didn't fix the problem, but I was able to resolve the issue by installing the requirements on the VM (via this line of code) so I think this should make it work